server/src/test/java/io/crate/integrationtests/CreateTableIntegrationTest.java

```diff
@@ -60,6 +72,7 @@ public void testCreatePartitionedTableIfNotExistsConcurrently() throws Throwable
     @Test
     public void test_allow_drop_of_corrupted_table() throws Exception {
         execute("create table doc.tbl (ts timestamp without time zone as '2020-01-01')");
+        ensureGreen();
```

Comment:
Is this really needed? Isn't yellow enough which is ensured by the table creation logic (wait for shards allocated before respond)

Reply:
I think not even yellow is needed. The test only requires the metadata.

I first had a mistake using `getMasterNodeInstance` instead of `getCurrentMasterNodeInstance` and used the `mapperService` from the indexService (which is absent  on a master-only node - leading to index-not-found errors). I added the `ensureGreen()` before realizing the mistakes and forgot to remove it.